### DISCUSSION TITLE
Icon

### DISCUSSION THREAD

**Discussion by westnordost** [Helium314/SCEE#597] [Ideas]
The SCEE icon always striked me as lazy.

How about something like this? (Just a quick mockup, didn't look at the correct icon sizes etc.)

![appicon_fancy-rect](https://github.com/user-attachments/assets/540002d9-be8b-4ff7-b5c8-0e71a7f89220)

The background and the "sign" metaphor stays the same as for SC to denote the relation to SC. The warning sign denotes that SCEE the app should really rather be used by people who know what they are doing, using the Vienna-convention sign rather than a generic yellow one (⚠️) because its more strongly related to street/traffic, most users are not US users (who have different signs) and because the yellow sign is a bit too unpleasant for the eye (too much " danger danger!")
For the content of the sign, I guess the magnifying glass is somewhat for OpenStreetMap, but it could also be e.g. He (for Helium), or Ee (for Expert Edition) or something else entirely.

**Comment by Helium314** [comment]
> The SCEE icon always striked me as lazy.

Oh it certainly is :grin: 

The icon looks like it's missing something though... maybe the SC guidepost inside the magnifying glass? Though maybe I first should look at the icon when it's app-icon-on-phone-display sized.

**Reply by westnordost** [reply]
Some quick sketches

![appicon_fancy-rect](https://github.com/user-attachments/assets/8c311ba7-38d0-499c-b176-7c21bf05fa00)
![appicon_fancy-rect2](https://github.com/user-attachments/assets/3059fce7-c257-4172-99f5-4ba78f8d6c39)
![appicon_fancy-rect3](https://github.com/user-attachments/assets/3d73d9c8-4fe6-4b34-867f-bdd41d7089fd)
![appicon_fancy-rect4](https://github.com/user-attachments/assets/966094e6-8f61-4432-9612-6241e691a8b2)
![appicon_fancy-rect5](https://github.com/user-attachments/assets/71c472b5-5342-401b-ad1b-3b5c2a02e376)
![appicon_fancy-rect6](https://github.com/user-attachments/assets/5f7e8a08-5681-4c74-96b9-6a1a3477f019)
![appicon_fancy-rect7](https://github.com/user-attachments/assets/107676cc-6873-4487-b9a4-54fae844e5a4)

But now, it's a warning of streetcomplete? 🤔

**Reply by westnordost** [reply]
![appicon_fancy-rect8](https://github.com/user-attachments/assets/a90b083b-6c2a-402d-8c22-ebdc0a07156f)

Or maybe it is not a warning, but a "yield", ... because... users should ... yield and take a moment to ensure they know what they are doing? (And because the SC icon has more volume at the top).

I guess the US-style warning sign would also work, shape-wise

**Reply by Helium314** [reply]
Actually I really like the last one!

**Reply by mnalis** [reply]
- (ref. https://github.com/Helium314/SCEE/pull/599)

**Comment by nobkd** [comment]
Just a side note, as it's nothing urgent: The SCEE icon is still the old one in F-Droid.
Maybe it has to be changed manually? :shrug:

**Reply by mnalis** [reply]
thanks for reporting! I've made PR https://github.com/Helium314/SCEE/pull/638 which should fix that
